infra/templates/az-hello-world/README.md

```diff
@@ -0,0 +1,112 @@
+# The Azure Hello World Pattern
+
+The `az-hello-world` template is intended to be a reference for running a single public Linux Container within an Azure Application Service Plan.
+
+> *Have you completed the quick start guide? Deploy your first infrastructure as code project with Cobalt by following the [quick-start guide](https://github.com/microsoft/cobalt/blob/master/docs/2_QUICK_START_GUIDE.md).*
+
+## Use-Case
+
+This particular template creates an Azure environment with our smallest infrastructure footprint and is the recommended template highlighted in our [quick-start guide](https://github.com/microsoft/cobalt/blob/master/docs/2_QUICK_START_GUIDE.md). It's a practical first step.
+
+## Provisioned Resources
+
+This deployment creates the following:
+
+ 1. Azure Resource Group
+ 2. Linux App Service Plan
+ 3. App Service Container w/ public i.p.
+
+> ![image](https://user-images.githubusercontent.com/10041279/66533295-0fc64b80-ead8-11e9-96c0-ea5441cde854.png)
+
+## Intended audience
+
+Application developer that is brand new to Cobalt templating and it's *Cobalt Infrastructure Template* (CIT) developer workflow.
+
+## Prerequisites
```

Comment:
Can you instead link to the getting started guide? Looks like the same info

Reply:
In the spirit of brevity, I agree.